node/src/main/scala/coop/rchain/node/encode/JsonEncoder.scala

```diff
@@ -78,6 +84,86 @@ object JsonEncoder {
   implicit val encodeConnective: Encoder[Connective] = deriveEncoder[Connective]
   implicit val encodeAlwaysEqual: Encoder[AlwaysEqual[BitSet]] =
     Encoder.encodeUnit.contramap[AlwaysEqual[BitSet]](_ => ())
+
+  // FIXME blake2b512Random encode. Question Is that really neccessary?
```

Comment:
We need this codec for definition but it does nothing in the resulting binary. Maybe to rename it as _dummy_?